src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -2204,6 +2204,23 @@ export function registerTerminalActions() {
 			clearShellFileHistory();
 		}
 	});
+	registerAction2(class extends Action2 {
+		constructor() {
+			super({
+				id: TerminalCommandId.ShowQuickFixes,
+				title: { value: localize('workbench.action.terminal.showQuickFixes', "Show Terminal Quick Fixes"), original: 'Show Terminal Quick Fixes' },
```

Comment:
'Terminal Quick Fix...'?

![image](https://user-images.githubusercontent.com/2193314/204362129-f2f4a2a2-b31b-45a6-8396-0bb2af046909.png)

Reply:
IMO, that should have a verb since it's a command